internal/tuf/tuf.go

```diff
@@ -184,6 +183,16 @@ func (d *Delegations) AddKey(key *Key) {
 	d.Keys[key.KeyID] = key
 }
 
+func (d *Delegations) RemoveKey(keyID string) error {
+	if _, ok := d.Keys[keyID]; !ok {
+		return fmt.Errorf("key %s, is not in this rule", keyID)
```

Comment:
Not sure this needs to error if the key isn't in the file.

Reply:
Fixed